backend/inspirehep/editor/errors.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from invenio_rest.errors import RESTException
+
+
+class EditorRevertToRevisionError(RESTException):
+    code = 503
```

Comment:
more appropriate is a ``400`` code instead, ``503`` service unavailable is a bit missleading.